src/tools/perf/perftest.c

```diff
@@ -726,14 +729,32 @@ static ucs_status_t parse_test_params(ucx_perf_params_t *params, char opt,
     }
 }
 
+static ucs_status_t adjust_test_params(perftest_params_t *params,
+                                       const char *error_prefix)
+{
+    test_type_t *test = &tests[params->test_id];
+
+    if (params->test_id == TEST_ID_UNDEFINED) {
+        ucs_error("%smissing test name", error_prefix);
```

Comment:
it seems it will be printed as `"command linemissing test name"`
should it be:
```
ucs_error("%s: missing test name", error_prefix);
```
or smth like that?